core/frontend/src/extension/providers/ExtensionServiceClient.ts

```diff
@@ -1,184 +0,0 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-import { AccessToken, Guid } from "@itwin/core-bentley";
-
-import { request, RequestOptions } from "../../request/Request";
-
-/** Structure of extensions from the ExtensionService
- * @internal
- */
-export interface ExtensionMetadata {
-  contextId: string;
-  extensionName: string;
-  version: string;
-  files: FileInfo[];
-  uploadedBy: string;
-  timestamp: Date;
-  status: ExtensionUploadStatus;
-  isPublic: boolean;
-}
-
-interface ExtensionUploadStatus {
-  updateTime: Date;
-  status: string;
-}
-
-interface FileInfo {
-  url: string;
-  expires: Date;
-  checksum: string;
-}
-
-/**
- * Client for querying, publishing and deleting iModel.js Extensions.
- *
- * The `imodel-extension-service-api` OIDC scope is required for all operations and the `imodel-extension-service:modify` is
- * required for modification operations (modify, publish, and deleting).
- * @alpha
- */
-export class ExtensionClient {
```

Comment:
was this being exposed in the api for core-frontend in a barrel file?

Reply:
yup